.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think ide settings are typically excluded from version control - we could add this file to `.gitignore`?

Reply:
done